### PR TITLE
Implemented Onboarding Animations

### PR DESCRIPTION
**Pull request by ReadeP**
Implemented Lottie for animations during onboarding.

## Overview

Implemented Lottie animations

## Changes Made

Replaced imageView with Lottie animationView.

## Test Coverage

Manual testing

## Screenshots (delete if not applicable)

<details>

  <summary>Screen Shot Name</summary>

  
  

</details>

### CODE REVIEW

File: Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift [hunk 1]
```diff
@@ -7,16 +7,17 @@
 //
 
 import UIKit
+import Lottie
```
Comment (kevinchan159) [inline Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift]: can we alphabetize

File: Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift [hunk 2]
```diff
@@ -7,16 +7,17 @@
...
 
 class OnboardingInfoViewController: OnboardingViewController {
 
     private let stackView = UIStackView()
-    private let imageView = UIImageView()
+    private let animationView = AnimationView()
     private let nextButton = UIButton()
-    private let onboardingImage: UIImage?
+    private let animation: String
```
Comment (kevinchan159) [inline Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift:18]: lets move this to next line to separate view vars from data vars

**Review by kevinchan159 — APPROVED** [review]
⭐️

File: Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift [hunk 3]
```diff
@@ -37,6 +38,14 @@ class OnboardingInfoViewController: OnboardingViewController {
         }
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        DispatchQueue.main.async {
```
Comment (whoiswillma) [inline Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift:44]: Why delay the animation play?

**Review by whoiswillma — APPROVED** [review]

**Review by whoiswillma — APPROVED** [review]

File: Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift (hunk 3 above)
Comment (ReadeP) [reply Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift:44]: Ran into a bug where the animation would not play when the next viewController was presented in the pageViewController if we do not do it this way.

**Pull request merged**